game/models.py

```diff
@@ -1,67 +1,70 @@
-from django.db import models
 from datetime import datetime
 
+from django.db import models
+
+
 class SingletonModel(models.Model):
     class Meta:
         abstract = True
 
     def save(self, *args, **kwargs):
         self.__class__.objects.exclude(id=self.id).delete()
-        super(SingletonModel, self).save(*args, **kwargs)
+        super().save(*args, **kwargs)
 
     @classmethod
     def load(cls):
         try:
             return cls.objects.get()
-        except cls.DoesNotExist:
+        except cls.DoesNotExist:        # todo: does this work?
```

Comment:
Copied this from a standard Django singleton example

Reply:
I'd prefer to raise it explicitly via 
```
from django.core.exceptions import ObjectDoesNotExist

...
try:
    ...
except ObjectDoesNotExist:
    ...
```

Reply:
Btw. I think you probably do not need a singleton (too much java I suppose?^^)